stable/gocd/README.md

```diff
@@ -214,6 +214,33 @@ $ helm install --name gocd-app --set server.persistence.existingClaim=PVC_NAME s
 | `agent.persistence.subpath.homego`            | The /home/go path on Persistence Volume              | `homego`             |
 | `agent.persistence.subpath.dockerEntryPoint`  | The /docker-entrypoint.d path on Persistence Volume  | `scripts`            |
 
+## SSH keys
+
+For accessing repositories over SSH, you need to add SSH keys to the server and the agent.
+
+Generate a new keypair, fetch the host key for the [host] you want to connect to and create the secret. The secret is
+structured to hold the entire contents of the .ssh folder on the GoCD server and agent.
+
+```bash
+ssh-keygen -t rsa -b 4096 -C "user@example.com" -f gocd-ssh -P ''
+
+ssh-keyscan [host] > gocd_known_hosts
+
+kubectl create secret generic gocd-ssh \
+    --from-file=id-rsa=gocd-ssh \
+    --from-file=id-rsa.pub=gocd-ssh.pub \
```

Comment:
@or1can  Could you change `id-rsa` to `id_rsa` (with an underscore) for the private and public keys? The ssh client picks up names with underscores by default, and the hyphen might confuse users.